restapi/delete-blocks.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Delete blocks of data'
+description: 'This page explains how to delete blocks of data.'
+sidebarTitle: 'Delete blocks'
+---
+
+After data ingestion, Axiom stores events in blocks of up to 65,536 events. You cannot delete individual events after you send them to Axiom, but you can delete entire blocks of data. With the `/v2/datasets/_apl/delete` endpoint, you can specify an APL query and delete all blocks that contain matching events.
```

Comment:
"to 65,536 events" Do we need to be this specific (expect impl could change), or would "tens of thousands of events" be ok? I actually think there could be scenarios where it’s less than that (based on time), but we should explore that offline.

Reply:
Agreed! Suggested a change below.